client/src/components/MapLegend.vue

```diff
@@ -50,11 +55,11 @@ export default {
      * This is called when the component is mounted and any time the incoming
      * bucketMap is updated.
      */
-    createLegend() {
+    createLegend(): void {
```

Comment:
Do you know if it's a best practice to put `void` for the return type here? We might have a lot of methods that don't return anything so we should be consistent (such as the below arrow function in a `forEach`).